v2/data/retry_rcc/windows.json

```diff
@@ -1,28 +1,27 @@
 {
-	"working_directory": "/tmp/outputdir",
-	"results_directory": "/tmp/results",
-	"log_directory": "/tmp/logs",
-	"environment": {
-		"rcc_binary_path": "C:\\Users\\vagrant\\AppData\\Local\\Microsoft\\WindowsApps\\rcc.exe",
-		"scheduler_robot_yaml_path": "C:\\robotmk\\v2\\data\\scheduler_rcc\\robot.yaml"
-	},
+	"working_directory": "C:\\tmp\\output",
+	"results_directory": "C:\\tmp\\results",
 	"suites": {
 		"test": {
-			"execution_interval_seconds": 10,
-			"robot_target": "C:\\robotmk\\v2\\data\\retry_suite\\/tasks.robot",
-			"variants": [
-				{
-					"variablefile": null,
-					"argumentfile": null
-				},
-				{
-					"variablefile": "C:\\robotmk\\v2\\data\\retry_suite\\/retry_variables.yaml",
-					"argumentfile": null
-				}
-			],
-			"retry_strategy": "incremental",
-			"robot_yaml_path": "C:\\robotmk\\v2\\data\\retry_rcc\\/robot.yaml",
-			"session": null
+			"robot_framework_config": {
+				"robot_target": "C:\\robotmk\\v2\\data\\retry_suite\\tasks.robot",
+				"variable_file": "C:\\robotmk\\v2\\data\\retry_suite\\retry_variables.yaml",
```

Comment:
How this translate into the use case, which we discussed with Simon:
After trying a test, it should be retried with a different set of variable files?

Reply:
Apparently, we misunderstood. This is not a use case we want to support.

Reply:
I am actually very much in favor of this. Retries should not really be done with a different configuration :+1: